src/ui/view/dataverse/component/generalMetadata/generalMetadataValue.tsx

```diff
@@ -0,0 +1,37 @@
+import { useTranslation } from 'react-i18next'
+import type { ItemGeneralMetadata } from '@/ui/view/dataverse/types'
+import './generalMetadata.scss'
+import './i18n/index'
+import { DateInterval } from './dateInterval'
+
+const isPeriodGeneralMetadata = (
+  metadata: ItemGeneralMetadata
+): metadata is ItemGeneralMetadata & { property: 'period'; value: [string, string] } =>
+  metadata.property === 'period'
+
+const isGeneralMetadata = (
+  metadata: ItemGeneralMetadata
+): metadata is Omit<ItemGeneralMetadata, 'value'> & { value: string } =>
+  metadata.property !== 'tags'
```

Comment:
I'm sorry but I still can't understand this type guard.

In [dataspace.tsx](https://github.com/okp4/dataverse-portal/pull/175/files#diff-eb3fbf240f42b523582d0f723848f398521bef291e50fbeb418d2f18fd2bb152), you added the `category` property to the `tags` object and declared its value with `generalMetadata`, that is a good point.

But, here, you say everything that is not `tags` is `GeneralMetadata`.

So, what type of metadata do `tags` belong to?

Reply:
You are right, `tags` must belong to general metadata.  I updated it in [818e093](https://github.com/okp4/dataverse-portal/pull/175/commits/818e0933963bb263e94957728966ca1687998c86), by reusing the existing narrowing function from the PR #174, that has been recently merged.
Here is the one from #174 that seems correct to me:
```ts
export const isGeneralMetadataWithIcon = (
  metadata: ItemGeneralMetadata
): metadata is Omit<ItemGeneralMetadata, 'value'> & { value: string } =>
  metadata.property !== 'tags' &&
  metadata.category === 'generalMetadata' &&
  isPropertyWithIcon(metadata.property)
  ```

Reply:
@0xLodz 
But, if you say _"tags must belong to general metadata"_, so why the type guard does not include them? (`metadata.property !== 'tags'`)

Reply:
Gotcha. The `metadata.property !== 'tags'` doesn't make sense here.
I amended the [previous commit](https://github.com/okp4/dataverse-portal/pull/175/commits/9767cbaadd5eb25a0cf867ee042ddb9c04ba0477). Sorry for the misunderstanding.